eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -1,6 +1,3 @@
-Import-Module "${PSScriptRoot}/modules/ChangeLog-Operations.psm1"
-. (Join-Path $PSScriptRoot SemVer.ps1)
-
 $SDIST_PACKAGE_REGEX = "^(?<package>.*)\-(?<versionstring>$([AzureEngSemanticVersion]::SEMVER_REGEX))"
```

Comment:
Given we are using this SEMVER_REGEX directly here it might be best to keep the import on SemVer.ps1 directly in this file.